pkg/controller/directvolumemigration/rsync.go

```diff
@@ -1645,22 +1736,45 @@ func (req rsyncClientPodRequirements) getRsyncClientPodTemplate() corev1.Pod {
 			},
 		},
 		TerminationMessagePolicy: corev1.TerminationMessageFallbackToLogsOnError,
-		Command:                  rsyncCommand,
+		Command:                  rsyncContainerCommand,
 		Ports: []corev1.ContainerPort{
 			{
 				Name:          DirectVolumeMigrationRsyncClient,
 				Protocol:      corev1.ProtocolTCP,
 				ContainerPort: int32(22),
 			},
 		},
-		VolumeMounts: volumeMounts,
+		VolumeMounts: rsyncVolumeMounts,
 		SecurityContext: &corev1.SecurityContext{
 			Privileged:             &isPrivileged,
 			RunAsUser:              &runAsUser,
 			ReadOnlyRootFilesystem: &trueBool,
 		},
 		Resources: req.resourceReq,
 	})
+
+	// append stunnel container
+	containers = append(containers, corev1.Container{
+		Name:                     DirectVolumeMigrationStunnel,
+		Image:                    req.image,
+		TerminationMessagePolicy: corev1.TerminationMessageFallbackToLogsOnError,
+		Command:                  stunnelContainerCommand,
+		Ports: []corev1.ContainerPort{
+			{
+				Name:          DirectVolumeMigrationStunnel,
+				Protocol:      corev1.ProtocolTCP,
+				ContainerPort: int32(2222),
+			},
+		},
+		VolumeMounts: stunnelVolumeMounts,
+		SecurityContext: &corev1.SecurityContext{
+			Privileged:             &isPrivileged,
+			RunAsUser:              &runAsUser,
+			ReadOnlyRootFilesystem: &trueBool,
```

Comment:
What happened to the `dropCapabilities` fields present in the previous template?